### PR TITLE
Brain

### PR DESCRIPTION
**Pull request by cmedinilla**

### CODE REVIEW

File: scripts/feedback.js [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+const github = require('./utils/github');
+const slack = require('./utils/slack');
+// const brain = require('./utils/brain');
 const json = JSON.parse(fs.readFileSync('config.json', 'utf8'));
 
 module.exports = function(robot) {
-    robot.respond(/feedback (.*)/i, function(msg){
-        var today = new Date();
-        feedback = msg.match[1];
+    robot.respond(/ (feedback|f|queja) (.*)/i, msg => {
```
Comment (Xotl) [inline scripts/feedback.js]: isn't there an extra space between  `/` & `(feedback`?

File: scripts/feedback.js [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+        const feedback = msg.match[2];
+        robot.brain.set(msg.message.user.id, feedback);
+        msg.reply(`This is your feedback: ${feedback}, are you ok? y/n`);
+    });
+
+    robot.respond(/(y|Yes|Y|yes)/i, msg => {
```
Comment (Xotl) [inline scripts/feedback.js]: if the regexp is insensitive then is not needed then the `yY` & `yes Yes`.

The same for line `25`.

**Review by Xotl — APPROVED** [review]
Looks good, but can you check the comments?

**Comment by cmedinilla** [comment]
Fixed i added another commits to add your feedback

**Review by Xotl — APPROVED** [review]
LGTM

**Pull request merged**